docs/rest.md

```diff
@@ -240,6 +245,8 @@ Sample result:
     * `update_available`: if present and not empty, it means the snap can be
       updated to the version specified as a value to this entry.
     * `channel`: which channel the package is currently tracking.
+    * `prices`: if present the snap needs to be purchased before installing. It's
```

Comment:
if we're going with `buy` as the verb, here we want “bought” instead of “purchased”.